File: Day_2_Q7.java
import java.util.*;

public class Solution {
    public int[] solution(int []arr) {
        int[] answer = {};
        
        int a  = -1;
        ArrayList<Integer> arr1 = new ArrayList<Integer>();
        
        for(int idx: arr){
            if (a != idx) {
                arr1.add(idx);
            }
            a= idx;
        }
        answer = new int[arr1.size()];
        for(int i=0; i<answer.length; i++){
            answer[i] = arr1.get(i);
            //System.out.println(answer[i]);
        }
        
        return answer;
        
        
    }

File: Day_2_Q7.py
# Q7 Answer Template

def solution(arr):
    answer = []
    answer.append(arr[0]) # 첫 문자 넣기.
    
    for i in range(1, len(arr)):
        flag = False
        
        if not flag and arr[i-1] != arr[i]:
            answer.append(arr[i])
            flag = True
    return answer

arr = [1,1,3,3,0,1,1]
#arr = [4,4,4,3,3]
answer = solution(arr)
print(answer)